db/migrate/20230329175222_create_post.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+
+      t.timestamps
+    end
```

Comment:
- [ ] please have a look at the ERD diagram as you can see the Post Table should have a column called likes_counter as well.
![afbeelding](https://user-images.githubusercontent.com/39240820/228655448-a9f9542e-2d2d-4f12-a20f-a10f270769d8.png)